### ISSUE TITLE
Scammer

### ISSUE THREAD

**Issue by kasperinjo** [dropout1337/Discord-Token-Creator#35]
You have 268 stars and about 440 followers, yet didn't add email or phone verification. This guy is a scammer.

**Comment by treehousekingcomic** [comment]
Of course, what do you expect with any repository for token generating

**Comment by lkeld** [comment]
hes not wrong just do it yourself nerd

**Comment by kasperinjo** [comment]
Stfu bitch you are not better

**Comment by lkeld** [comment]
> Stfu bitch you are not better

🥱🥱🥱

**Comment by Vinyzu** [comment]
Nice nontoxic Community 😄.
Everyone in the DiscordBotting scene is so greatful for open-source code they get for free!

**Comment by treehousekingcomic** [comment]
> Nice nontoxic Community 😄.
> Everyone in the DiscordBotting scene is so greatful for open-source code they get for free!

Except most of not all aren't actually giving any open source for free

**Comment by Vinyzu** [comment]
> > Nice nontoxic Community 😄.
> > Everyone in the DiscordBotting scene is so greatful for open-source code they get for free!
> 
> Except most of not all aren't actually giving any open source code for free

Ye, and most are just skidding sauce from github, selling it, and then acting like their some kind of business man hahaha
